File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/EncryptedCredentialsFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.passport.EncryptedCredentials;

public class EncryptedCredentialsFilterValidator extends FilterValidator<EncryptedCredentials> {

    public EncryptedCredentialsFilterValidator(CustomValidator<EncryptedCredentials> validator) {
        super(EncryptedCredentials.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/EncryptedCredentialsFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.telegram.bot.api.model.objects.passport.EncryptedCredentials
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class EncryptedCredentialsFilterValidator(validator: Validator<EncryptedCredentials>) : FilterValidator<EncryptedCredentials>(EncryptedCredentials::class, validator)
